wcc-contentful-app/app/controllers/wcc/contentful/app/contact_form_controller.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+class WCC::Contentful::App::ContactFormController < ApplicationController
+  def create
+    id = params['id']
+    raise ArgumentError, 'missing form ID' unless id
+
+    form_model = WCC::Contentful::Model.find(id)
+    to_email = form_model.notificationEmail
```

Comment:
We're assuming that the model is of a certain content type here, because we're calling `notificationEmail` on it.  It can't be just any content type.

Please change line 8 to use that content type to do a find, which makes that assertion more clear and gives a better error message.

Reply:
Good point. This was a poor copy paste. Now I'm using `WCC::Contentful::Model::SectionContactForm` to make this call.